ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java

```diff
@@ -33,15 +32,14 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
-    public void EndVisitTest() throws Exception {
+    public void endVisitTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(patient.identifier);
         ClinicianFacingPatientDashboardPage clinicianFacingPatientDashboardPage = findPatientPage.clickOnFirstPatient();
         PatientVisitsDashboardPage patientVisitsDashboardPage = clinicianFacingPatientDashboardPage.goToRecentVisits();
         patientVisitsDashboardPage = patientVisitsDashboardPage.endVisit();
-        assertNull(patientVisitsDashboardPage.getActiveVisit());
```

Comment:
> Have tested this by ignoring and it passed,

After a deep simulation with my virtual remote server on  CI Vs Bamboo, I discovered that by just removing @ignore annotation, the test passes with CI But fails with Bamboo. Ideally I trust bamboo test result than CI report.  Though the element that is consumed by  `getActiveVisit()` method passes with [startVist](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/285dcc90731099bd2d9368097428a21b3c8bcca2/ui-tests/src/test/java/org/openmrs/reference/StartVisitTest.java#L40) test when `assertNotNull()` is applied But the element becomes no longer attached to the DOM  when `assertNull()` is applied. Different element looks ups (`id(), xpath(), linkText()`) have a similar case. 

>do you think this change is necessary

 Its from the reason above that we rather use `assertTrue()` function since it achieves the same goal in this case